recipes/nvcc/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "nvcc" %}
+{% set version = environ.get("CUDA_VER", "9.2") %}
+
+package:
+  name: "{{ name }}"
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+outputs:
+  - name: "{{ name }}_{{ target_platform }}"
+    version: "{{ version }}"
+    script: install_nvcc.sh
+    build:
+      script_env:
+        - CUDA_HOME
+      ignore_run_exports:
+        - libgcc-ng
+      run_exports:
+        strong:
+          - cudatoolkit {{ version }}|{{ version }}.*
+    requirements:
+      host:
+        # Needed to symlink libcuda into sysroot libs.
+        - {{ compiler("c") }}
+      run:
+        - {{ compiler("c") }}
+        # Host code is forwarded to a C++ compiler
+        - {{ compiler("cxx") }}
+    test:
+      files:
+        - test.cu
+      commands:
+        # Verify the symlink to the libcuda stub library exists.
+        - test -f "$(${CC} --print-sysroot)/lib/libcuda.so"
+        # Verify the activation scripts are in-place.
+        {% for state in ["activate", "deactivate"] %}
+        - test -f "${PREFIX}/etc/conda/{{ state }}.d/{{ PKG_NAME }}_{{ state }}.sh"
+        {% endfor %}
+        # Try using the activation scripts.
+        - source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh
+        - if [ -z ${CUDA_HOME+x} ]; then echo "CUDA_HOME is unset after activation" && exit 1; else echo "CUDA_HOME is set to '$CUDA_HOME'"; fi
+        - source ${PREFIX}/etc/conda/deactivate.d/{{ PKG_NAME }}_deactivate.sh
+        - if [ -z ${CUDA_HOME+x} ]; then echo "CUDA_HOME is unset after deactivation "; else echo "CUDA_HOME is set to '$CUDA_HOME' after deactivation" && exit 1; fi
+        # Try building something.
+        - source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh
+        - nvcc test.cu
+        - source ${PREFIX}/etc/conda/deactivate.d/{{ PKG_NAME }}_deactivate.sh
+    about:
+      home: https://github.com/conda-forge/nvcc-feedstock
+      license: BSD 3-Clause
+      license_file: LICENSE.txt
+      summary: A meta-package to enable the right nvcc.
+
+about:
+  home: https://github.com/conda-forge/nvcc-feedstock
+  license: BSD 3-Clause
+  license_file: LICENSE.txt
+  summary: A meta-package to enable the right nvcc.
+
+extra:
+  recipe-maintainers:
+    - jakirkham
```

Comment:
Please add me as well.

Reply:
Sure would be happy to have your help here. 🙂 Added you in the last commit.